bridge_adaptivity/module/policies/policy_points_earned.py

```diff
@@ -0,0 +1,12 @@
+from .base import BaseGradingPolicy
+
+
+class PointsEarnedGradingPolicy(BaseGradingPolicy):
+    """GRadding policy class calculate grade based upon users earned points."""
+
+    public_name = 'Points earned'
+    internal_name = 'points_earned'
+
+    def _calculate(self):
+        trials_count, points_earned = self._get_points_earned_trials_count()
+        return float(points_earned) / max(self.policy.threshold, trials_count)
```

Comment:
DevisionbyZero impossible threshold positive integer field

Reply:
```threshold = models.PositiveIntegerField(blank=True, default=0, help_text="Grade policy: 'Q'")```
threshold could be equal to zero... I copied this line from Collection model

Reply:
but, yes, in general, it can't raise this exception because trials_count will be min 1.